packages/account/tests/SmartAccountV2-Abstract-Paymaster.local.spec.ts

```diff
@@ -1,102 +1,86 @@
-import { EntryPoint, EntryPoint__factory } from "@account-abstraction/contracts";
-import { Wallet, ethers } from "ethers";
-
-import {
-  SmartAccount_v200,
-  SmartAccountFactory_v200,
-  SmartAccount_v200__factory,
-  SmartAccountFactory_v200__factory,
-  ECDSAOwnershipRegistryModule_v100__factory,
-} from "@biconomy/common";
-
-import { BiconomySmartAccountV2 } from "../src/BiconomySmartAccountV2";
-import { ChainId } from "@biconomy/core-types";
-import { ECDSAOwnershipValidationModule } from "@biconomy/modules";
-import { BaseValidationModule } from "@biconomy/modules";
-import { ECDSAOwnershipRegistryModule_v100 } from "@biconomy/common";
-import { BiconomyPaymaster } from "@biconomy/paymaster";
-import { Hex } from "viem";
-
-const provider = new ethers.providers.JsonRpcProvider("http://127.0.0.1:8545");
-const signer = provider.getSigner();
+// import { EntryPoint, EntryPoint__factory } from "@account-abstraction/contracts";
```

Comment:
Commented until we can use this right? https://github.com/bcnmy/scw-contracts/tree/main/test/bundler-integration